tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -33,11 +33,9 @@
 @pytest.mark.dimensions("us-gov-west-1", "c5.xlarge", "ubuntu1804", "slurm")
 @pytest.mark.dimensions("eu-west-1", "g3.8xlarge", "alinux2", "slurm")
 @pytest.mark.dimensions("eu-west-1", "g3.8xlarge", "centos7", "slurm")
-@pytest.mark.dimensions("eu-west-1", "g3.8xlarge", "centos8", "slurm")
```

Comment:
Thank you for taking care of the integration tests. We will just need to remove all the annotations